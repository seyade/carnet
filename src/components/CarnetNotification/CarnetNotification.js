import React, { Component } from 'react';

class CarnetNotification extends Component {
  constructor(...props) {
    super(...props);

    this.state = {
      notificationText: ''
    };
  }

  render() {
    return (
      <div className="carnet-notification">
        <p>{this.props.notificationText}</p>
      </div>
    );
  }
}

export default CarnetNotification;
