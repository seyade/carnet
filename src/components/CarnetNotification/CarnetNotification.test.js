import React from 'react';
import ReactDOM from 'react-dom';
import { mount, shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import CarnetNotification from '../CarnetNotification/CarnetNotification';

describe('CarnetNotification', () => {

  test('renders CarnetNotification as intended', () => {
    const component = renderer.create(<CarnetNotification />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
