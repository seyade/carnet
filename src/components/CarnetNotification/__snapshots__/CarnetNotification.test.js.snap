exports[`CarnetNotification renders CarnetNotification as intended 1`] = `
<div
  className="carnet-notification">
  <p />
</div>
`;
